requirements/dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
-# by the following command:
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
```

Comment:
@biyeun this change is unexpected. What version of `pip-tools` do you have? Should be 7.3.0

Reply:
ah i had `6.8.0`